src/main/java/com/cloudogu/scm/review/comment/service/CommentService.java

```diff
@@ -123,8 +138,10 @@ public void modifyComment(String namespace, String name, String pullRequestId, S
     PermissionCheck.checkModifyComment(repository, rootComment);
     Comment clone = rootComment.clone();
     rootComment.setComment(changedComment.getComment());
+    rootComment.setMentionUserIds(mentionMapper.extractMentionsFromComment(changedComment.getComment()));
```

Comment:
Maybe this could be merged with the postMentionEventIf... method into a `handleMentions` method or something alike